sentry_sdk/integrations/spark/spark_driver.py

```diff
@@ -106,99 +106,101 @@ def process_event(event, hint):
 
 
 class SparkListener(object):
-    def onApplicationEnd(self, applicationEnd):
+    def onApplicationEnd(self, applicationEnd):  # noqa: N802,N803
```

Comment:
Maybe apply this to the whole class or module/file?
Doing it per method is very noisy :(

Also could we have a note like "# noqa: ...; ignore camelCase in methods names and attributes because this code is inherited from somewhere..."

Reply:
I am not sure how to specify it on a class level. I would like to avoid doing it on a module level for sure asssuming we do not restructure the modules.